integration/js/pages/AppPage.js

```diff
@@ -124,6 +126,18 @@ class AppPage {
     await clickElement(this.driver, authenticateButton);
   }
 
+  async openAdditionalOptions() {
```

Comment:
This is an extra step for the integration tests 😅 @ltrung any preference?
 This PR adds a modal for the additional options, for choosing an option do I have to open the additional options and then choose something? If so, these I think are currently choosing options based on tests and for example, EventReporting test does not run as part of PR checks but as part of canaries. Can you once confirm whether all those current option choosing are handled?

Ref: https://github.com/aws/amazon-chime-sdk-js/blob/4a403f6a37ea84d16e06c84dbfa54a8fdcf0fc26/integration/js/steps/AuthenticateUserStep.js#L62-L77

Reply:
This makes may be UX better for end users but extra work for the integration test. I am fine to choose UX over small extra integration test work.

Reply:
Yes I checked the options selection buttons function on `AppPage` and confirmed all tests which called them (well, they all just call `AuthenticateUserStep` are wrapped in the open function). Though now I do see that the `selectRegion` is not in the right place as that should be outside additional options.